src/selena/services/models.py

```diff
@@ -159,6 +159,15 @@ class Service(models.Model):
         choices=AuthMethodChoices(),
         default=AuthMethodChoices.none.id,
     )
+    maas_intkey = models.CharField(
+        _('MaaS Integration Key'),
+        max_length=32,
+        blank=True,
+        null=True
+    )
+    sc_id = models.PositiveIntegerField(
+        _('Service Catalog ID'), blank=True, null=True
```

Comment:
again, SC uses US English form, let's keep it consistent throughout the ecosystem :P